net._86box._86Box.yaml

```
@@ -64,6 +68,7 @@ modules:
           - mkdir -p ${XDG_CONFIG_HOME}/86Box
           - exec 86Box --vmpath ${XDG_CONFIG_HOME}/86Box "$@"
     post-install:
+      - mkdir -p ${FLATPAK_DEST}/share/86Box
```

Extension mount directory must exist in the app.
```suggestion
      - mkdir -p ${FLATPAK_DEST}/share/86Box/roms
```

---

Ah, I did not realize this. Thanks!